src/utils/vehicle_util.lua

```diff
@@ -1,20 +1,22 @@
 vehicle_util = {}
 
 function vehicle_util.ROSControl(self, dt, acceleration, allowedToDrive, rotatedTime)
+    aiSteeringSpeed=0.0005
     if acceleration ~= nil and rotatedTime ~= nil and allowedToDrive ~= nil then
-        self.rotatedTime = rotatedTime
```

Comment:
The variables is more than less the steering angle, so the movement was jerky because of line 5 as it was jumping straight to the angle instead of now where the changing in small jumps

Reply:
That explains some strange behaviour Ciarán and I observed in one of the videos from the authors. Good job.

Reply:
Hi. We would be grateful if you could contribute this change upstream.

That would solve it for everyone, instead of just in your fork.

Reply:
We are currently testing it together with some changes to the TEB planner. Once it is verified we will be happy to share this change